### PR TITLE
Add Dropdown for Control Center

### PR DESCRIPTION
**Pull request by elfenheart**
Need to refactor once https://github.com/caskdata/cdap/pull/10001 is merged

Adding dropdown for Dashboard and Reports

### CODE REVIEW

File: cdap-ui/app/cdap/components/Header/ControlCenterDropdown/ControlCenterDropdown.scss [hunk 1]
```diff
@@ -14,8 +14,9 @@
...
  */
 
-@import '../DropdownStyling.scss';
-
-.metadata-dropdown {
-  @include dropdownstyling;
+.control-center.header-dropdown {
+  .dropdown-menu {
+    // offset to align the dropdown to the begining of the Control Center button
+    left: -16px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Header/ControlCenterDropdown/ControlCenterDropdown.scss:20]: nit: the `with-shadow` li element has a padding left and right of 15px. Should this be `-15px`?

**Review by ajainarayanan — COMMENTED** [review]
One minor comment. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/Header/ControlCenterDropdown/ControlCenterDropdown.scss (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Header/ControlCenterDropdown/ControlCenterDropdown.scss:20]: `-15px` was not quite aligned. I think with the border, it adds 1px offset.

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**